sys-cluster/glusterfs/glusterfs-4.1.5.ebuild

```diff
@@ -0,0 +1,247 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit autotools elisp-common python-single-r1 systemd user versionator
```

Comment:
s/versionator/eapi7-ver/, unless you find that all the eclasses here are
eapi 7 capable, in which case use that and drop versionator.

Reply:
I will check every eclass and will try to bump EAPI version. It will take some time.

Reply:
Easier to bump eclass real quick and just try to ebuild it. It will tell you
if any cannot.